docs/vocabulary.md

```diff
@@ -388,17 +388,64 @@ pipeline in corresponding documentation.
 | **Tekton**         | N/A       | Step       | Task     | Pipeline | Trigger | Resource (?)           |
 | **Zuul**           | N/A       | N/A        | Job      | Pipeline | Trigger | Node (?)               |
 
-### Build Stages
-
-The table below is an attempt to create a mapping of common names for build stages (the unit of work one degree smaller than a pipeline). While CI/CD Tools and Technologies generally give developers broad leeway in naming and implementing their build stages, agreeing upon some common terms will help develop more pluggable build pipelines.
-
-| Stage      | Semantics     | Aliases       | Inputs   | Outputs and Results  |
-|------------|---------------|---------------|----------|-----------------------|
-| **Build** | Set up the build workspace and provision build resources. Download, retrieve, assemble, and/or compile software into an executable and testable format. Download, retrieve, assemble, and/or compile documentation into a consumable format. | N/A | Source Code, Other Software Artifacts, Documentation | Executable Software, Compiled Documentation |
-| **Test** | Test, scan, verify, and lint software and documentation. | Verify | Executable Software, Compiled Documentation | Test/Lint/Scan Results, Test/Lint/Scan Reports, Test/Lint/Scan Coverage Reports |
-| **Release** | Package, version, sign, and publish software artifacts and documentation. | Publish | Software Artifacts, Documentation | Release Reports, Software Artifacts and Documentation: packaged, signed, and published to a repository. |
-| **Deploy** | Deploy software artificats and documentation to any environment other than the build environment. Verify successful deployment. | N/A   | Software Artifacts, Documentation | Software running in another environment. Documentation hosted in another environment. |
-| **Maintain** | Automatically update or upgrade previously built software. May include some or all of the previous stages. | Update, Upgrade | See Above | See Above |
+### Stages
+
+The list below is an attempt to create a mapping of common names for Stages, where a Stage is the unit of work one degree smaller than a Pipeline. While CI/CD Tools and Technologies generally give developers broad leeway in naming and implementing their Stages, agreeing upon some common terms will help develop more pluggable Pipelines.
```

Comment:
The terms "Stage" and "Pipeline" are not yet set as the generic words for these pieces. The effort of proposing a common vocabulary based on the terms in the table above is not yet performed. I'd expect that the terms Stage and Pipeline would get many votes though, so I wouldn't say it is all wrong, but you should probably add a comment to this section about that those names are not yet set.

Reply:
OK, I'll add that in.

Reply:
Will do.